docs/source/algorithms/SaveNexusGeometry-v1.rst

```diff
@@ -10,35 +10,60 @@
 Description
 -----------
 
-TODO: Enter a full rst-markup description of your algorithm here.
+Saves the Instrument geometry from the workspace to a Nexus file. Saves detector positions and rotations.
+Current version (1.0) does NOT save instrument shapes of instrument components. file is compliant to Nexus standards.
+For more information on the Nexus format, see https://www.nexusformat.org/
+
+the Instrument will be extracted from the specified workspace, and written to the specified location.
+
+The (optional) H5 root group name is the parent group in which the Instrument and sample data are stored.
+If no name is given, the root group will have a default nme of 'entry'
 
 
 Usage
 -----
-..  Try not to use files in your examples,
-    but if you cannot avoid it then the (small) files must be added to
-    autotestdata\UsageData and the following tag unindented
-    .. include:: ../usagedata-note.txt
 
 **Example - SaveNexusGeometry**
 
 .. testcode:: SaveNexusGeometryExample
 
-   # Create a host workspace
-   ws = CreateWorkspace(DataX=range(0,3), DataY=(0,2))
-   or
-   ws = CreateSampleWorkspace()
+   # create sample workspace
```

Comment:
I think @DanNixon might have some suggestions for this section.

Reply:
@TakudzwaMakoni Have a look at the documentation for `SaveNexusProcessed` for an idea of how to write a doc test for a saver.

Reply:
Also not that the `testcleanup` block is an important thing to add. This is where you revert any changes you made to the filesystem.